.changeset/plenty-cows-train.md

```diff
@@ -0,0 +1,10 @@
+---
+"@jpmorganchase/uitk-lab": minor
+---
+
+Rename layer examples
```

Comment:
Do you think non-user facing changes should be added to the change log?

Reply:
That's a good point, I don't think they should so I've updated the change log

Reply:
Although, I've seen some libraries add documentation edits in to the change log

Reply:
@origami-z @joshwooding what do you reckon?

Reply:
Hmm, I think for now leaving them out is fine. Potentially in the future we can add them.